README.md

```diff
@@ -264,11 +264,9 @@ func (c *Controller) ShowAccount(ctx *gin.Context) {
 // @Accept  json
 // @Produce  json
 // @Param q query string false "name search by q"
-// @Success 200 {array} model.Account
+// @Response 200 {array} model.Account
```

Comment:
Why change to `@Response` tag?

Reply:
Refer to #692 , I added a @response tag, the same effect as @success and @failure.
There is already an example showing @success and @failure just above this one.